zephyr-mcumgr-python/src/return_types.rs

```diff
@@ -22,6 +22,28 @@ impl From<commands::fs::FileStatusResponse> for FileStatus {
     }
 }
 
+/// Return value of `MCUmgrClient.os_mcumgr_parameters`.
+#[gen_stub_pyclass]
+#[pyclass(frozen)]
+#[derive(Serialize)]
+pub struct MCUmgrParametersResponse {
```

Comment:
@coderabbitai Missing `export` in `lib.rs` for this struct. Please check this in the future. All public types must be exported otherwise they do not show up in `pdoc` documentation generator.